26.md

```diff
@@ -1,13 +1,13 @@
 
-NIP-26
+NIP-27
```

Comment:
Please take a look at the [list of PRs](https://github.com/nostr-protocol/nips/pulls), this one is taken too.